hack/dockerfile/install/containerd.installer

```diff
@@ -5,7 +5,8 @@
 # fixes or new APIs.
 : "${CONTAINERD_COMMIT:=814b7956fafc7a0980ea07e950f983d0837e5578}" # v1.3.4
 
-install_containerd() {
+install_containerd() (
+	set -e
```

Comment:
why here? this is supposed to be at top of files

Reply:
I did the `()` braces to run it in a sub-shell, so that the `set -e` wouldn't affect other scripts running in the same session. Not sure if that makes sense 😅